### PR TITLE
Decisions/Group 5

### PR DESCRIPTION
**Pull request by ColemanKobe**
This is the pull request from Group 5, where we made contributions primarily to "Decisions" but other aspects of the Code of Conduct as well, such as "Attendance," "Deadlines," and "Communication."

### CODE REVIEW

**Comment by enpuyou** [comment]
@ColemanKobe I see there are some merge conflicts in the file `conduct.md`.  Please resolve these before the PR is merged.

File: conduct.md
Comment (noorbuchi) [inline conduct.md:46]: Please be consistent about where you add a new bullet point. Specifically, it should either be always above the `TODO` or always below it. This would help other groups know if a section has already been completed or not.

File: conduct.md
Comment (noorbuchi) [inline conduct.md:72]: Please revise the order and the `TODO` to be consistent with other sections

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Can you please explain the purpose of these `=` signs and why they were added.

File: conduct.md
Comment (noorbuchi) [inline conduct.md:57]: Stay consistent with TODO order here as well

**Review by noorbuchi — CHANGES_REQUESTED** [review]
Hello, nice additions overall! I requested few changes for the purpose of consistency and clarification and it would be great if you could address them before this PR can be approved and merged. My other request would be to be more detailed in your PR message, specifically, what are the `other aspects of the Code of Conduct` that you made changes to?

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Formatting the names like this will put them all on the same line once Markdown is rendered, It would be great to add commas between names to make this section more readable.

File: conduct.md
Comment (ColemanKobe) [reply conduct.md]: I'm not sure how they were added.

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Suggestion: consider rephrasing this sentence to include "assigned work in general" instead of just bugs and technical issues. This is not a request and if you don't agree with this suggestion then that's completely fine.

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Please make sure that all names are next to each other. In this case `Janyl Jumadinova` is on a completely different line. Feel free to put the `TODO` before or after the name

File: conduct.md
Comment (noorbuchi) [inline conduct.md]: Efficient communication and communicating with the appropriate team member were mentioned twice in this list. It would also be great if you can explain what exactly do you mean by efficient communication

**Review by noorbuchi — CHANGES_REQUESTED** [review]
Hello, I noticed few other needed changes to this PR. Once a requested change has been added, please feel free to resolve the conversation where it was mentioned. Also, please don't forget to add a more descriptive PR message as I mentioned in my previous reviews. Thank you for all your work!

File: conduct.md
Comment (enpuyou) [inline conduct.md]: I think it would be better to not use contraction in a formal document like this; namely, "that is" instead of "that's"

**Review by enpuyou — COMMENTED** [review]
It looks fine to me overall. One question: have your group notified the other groups before editing the sections of attendance, deadline, and so on. It is noted in the decision section of the conduct that "members shall not make decisions that would affect another team without first notifying them"

**Review by noorbuchi — APPROVED** [review]
I think this PR looks good! Thank you for addressing the requested changes and the comments that I added. Please wait until other reviewers approve this PR before merging it. Nice work!

**Review by enpuyou — APPROVED** [review]
Looks good! The PR is ready to go. I would also like to request members from the other teams to review and give feedbacks.

File: conduct.md
Comment (PaigeCD) [inline conduct.md:72]: This should work just fine with what my group has for Communication without sounding too redundant.

**Review by PaigeCD — APPROVED** [review]

**Pull request merged**
